app/views/plans/_edit_details.html.erb

```diff
@@ -1,95 +1,162 @@
+        <%= f.label _('Name'), for: :data_contact, 'aria-label': _('Data Contact Name') %>
+        <%= f.text_field :data_contact, class: 'input-medium left-indent' %>
+      </div>
+      
+      <div class="form-input data-contact-info">
+        <%= f.label _('Email'), for: :data_contact_email, 'aria-label': _('Data Contact Email') %>
+        <%= f.email_field :data_contact_email, class: 'input-medium left-indent' %>
+        <span role="" id="data_contact_email_error" class="error-tooltip-right left-indent"></span>
       </div>
       
       <br />
       <br />
       <div class="form-input">
-        <div class="button-spacer">&nbsp;</div>
-        <%= submit_tag(_('Save'), class: 'form-submit') %>
+        <%= render partial: 'shared/accessible_submit_button',
+                   locals: {id: 'save-details-button', 
+                            val: _('Save'), 
+                            disabled_initially: true,
+                            classes: 'small-input-button left-indent',
+                            tooltip: _('You must provide a project title and any email addresses must be valid.')} %>
       </div>
     </fieldset>
   </div>
 
   <div class="edit-plan-right">
-    <fieldset class="standard">
-      <%= f.hidden_field(:visibility, value: visibility) %>
-      <span class="fa fa-question-circle"></span>
-      <div class="left-indent inline">
-        <%= f.label(:visibility, _('Plan Visibility')) %>
-        <div class="left-indent">(<%= _('Limited to finished plans') %>)</div>
-        <br />
-        <div>
-          <%= radio_button_tag(:vis, 0, !['publicly_visible', 'organisationally_visible'].include?(visibility),
-              class: 'right-indent',
-              id:'privately_visible') %>
-          <%= raw display_visibility('privately_visible') %>
-        </div>
-        <div>
-          <%= radio_button_tag(:vis, 0, 'organisationally_visible' == visibility,
-              class: 'right-indent',
-              id: 'organisationally_visible') %>
-          <%= raw display_visibility('organisationally_visible') %>
-        </div>
-        <div>
-          <%= radio_button_tag(:vis, 0, 'publicly_visible' == visibility,
-              class: 'right-indent',
-              id: 'publicly_visible') %>
-          <%= raw display_visibility('publicly_visible') %>
-        </div>
-      </div>
-    </fieldset>
-    
     <div class="bordered left-indent">
-      <div class="bordered"><strong><%= _('Plan Guidance Configuration') %></strong></div>
+      <h2><%= _('Plan Guidance Configuration') %></h2>
       <p><%= _('To help you write your plan, %{application_name} can show you guidance from a variety of organisations.') % {application_name: Rails.configuration.branding[:application][:name]} %></p>
-      <br />
-      <p><% _('Select up to 3 organisations to see their guidance.') %></p>
-      <% #@important_ggs.inspect %>
+      <p><%= _('Select up to 3 organisations to see their guidance.') %></p>
```

Comment:
Durring the call with UX on Wednesday, it was mentioned that 3 was an arbitrary limit due to spacing of the new guidance tabs within plan edit.  If more than 3 are needed, @JEK-III said he would be able to re-design the guidance display to display more, either through a second row of tabs or otherwise.